features/install-cli.feature

```diff
@@ -48,6 +48,20 @@ Feature: Command behavior of client side tools installation
     When I run `bundle exec vagrant service-manager install-cli docker --cli-version 111.222.333`
     Then the exit status should be 126
 
+    When I run `bundle exec vagrant service-manager install-cli docker`
+    And box is "adb"
```

Comment:
Have you tested this properly and do the expected counts match? Run tests against HEAD and check the assertion/scenario count for ADB and CDK. Then run against your pull request, you need two assertions more per box type